content/compose/environment-variables/interpolation.md

```diff
@@ -0,0 +1,305 @@
+---
```

Comment:
This file should be added in the [toc file ](https://github.com/docker/docs/blob/main/data/toc.yaml)